CDTDatastore/touchdb/TD_Database+Insertion.m

```diff
@@ -629,7 +640,8 @@ - (TDStatus)forceInsert:(TD_Revision*)rev
                                            parentSequence:sequence
                                                   current:current
                                                      JSON:json
-                                                 database:db];
+                                                 database:db
+                                                    error:NULL];
```

Comment:
Wouldn't we be better making use of the error to differentiate between db full errors and general db errors as done elsewhere?